src/Gamepage/Gamepage.js

```diff
@@ -67,20 +66,38 @@ export class Gamepage extends Component {
   render = () => {
     return (
       <div className="game-page" data-testid="game-page">
-        <div className="hints">
-          <h2><em>Hints</em></h2>
-
+        <div className="chat">
+          <h2>
+            <em>Chat</em>
+          </h2>
         </div>
         <div className="board">
           <div className="draw-board">
-            <h2><em>Draw Board</em></h2>
-          <div className="display-word">{this.splitDisplay()}</div>
+            <h2>
+              <em>Draw Board</em>
```

Comment:
We should probably remove `Draw Board` if we're not going to implement drawing by the due date

Reply:
True! It might be cumbersome if each player will have their own display, too...